package.json

```diff
@@ -61,11 +61,13 @@
 		"@types/glob": "^7.1.1",
 		"@types/mocha": "^5.2.6",
 		"@types/node": "^8.10.0",
+		"@types/sinon": "^7.0.13",
 		"glob": "^7.1.4",
 		"mocha": "^6.1.4",
+		"sinon": "^7.3.2",
```

Comment:
I don't see the use of this package `sinon` in the newly added code, maybe you don't need it?

Reply:
@samiyaakhtar it’s required for unit tests that I was working on for a later PR, should I remove it for this PR?